File: Java/Array/BJAlgo_2920.java
import java.io.BufferedReader;
import java.io.InputStreamReader;

public class BJAlgo_2920 {
    public static void main(String[] args) throws Exception {
        BufferedReader br = new BufferedReader(new InputStreamReader(System.in));

        String[] tmp = br.readLine().split(" ");

        boolean ascending = true;
        boolean descending = true;

        for(int i = 1; i < tmp.length; i++) {
            if(Integer.parseInt(tmp[i]) > Integer.parseInt(tmp[i - 1])) {
                descending = false;
            } else if(Integer.parseInt(tmp[i]) < Integer.parseInt(tmp[i - 1])) {
                ascending = false;
            }
        }

        if (ascending) {
            System.out.println("ascending");
        } else if (descending) {
            System.out.println("descending");
        } else {
            System.out.println("mixed");
        }
    }
}

File: Python/Array/BJAlgo_2920.py
from sys import stdin

# inputs = list(map(int, stdin.readline().split()))
# asc = sorted(inputs)
#
# if inputs == asc:
#     print('ascending')
# else:
#     desc = sorted(inputs, reverse=True)
#     if inputs == desc:
#         print('descending')
#     else:
#         print('mixed')


# fastcampus 문제 풀이
inputs = list(map(int, stdin.readline().split()))

ascending = True
descending = True

for i in range(1, 8):
    if inputs[i] > inputs[i - 1]:
        descending = False
    elif inputs[i] < inputs[i - 1]:
        ascending = False

if ascending:
    print('ascending')
elif descending:
    print('descending')
else:
    print('mixed')